bucket/scantailor-advanced.json

```
@@ -0,0 +1,34 @@
+{
+    "description": "A fork of ScanTailor that merges the features of the ScanTailor Featured and ScanTailor Enhanced versions, brings new ones and fixes.",
+    "homepage": "https://github.com/4lex4/scantailor-advanced",
+    "license": "GPL-3.0-only",
+    "version": "1.0.16",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/4lex4/scantailor-advanced/releases/download/v1.0.16/scantailor-advanced-1.0.16-win64.exe#!/dl.7z",
+            "hash": "ce3c1bae867ccaf020c759efad463ae0b937fac5ea3ef8fc4ce559ddd1ae8fc9"
+        },
+        "32bit": {
+            "url": "https://github.com/4lex4/scantailor-advanced/releases/download/v1.0.16/scantailor-advanced-1.0.16-win32.exe#!/dl.7z",
+            "hash": "c73a46f3a2f3d99b434a8724c33d7b9745fa90253cdae8640dd99fac9793b032"
+        }
+    },
+    "shortcuts": [
```

- No bin
- Remove unneeded files

```suggestion
    "post_install": "Remove-Item \"$dir\\`$*\" -Recurse",
    "bin": "scantailor.exe",
    "shortcuts": [
```

https://github.com/scantailor/scantailor/blob/master/CommandLine.cpp
https://github.com/4lex4/scantailor-advanced/blob/master/CommandLine.cpp

---

😂Both original scantailor and another fork have a `scantailor-cli` executable as `bin`, but scantailor-advanced doesn't.